components/core/ApplicationUserControls.js

```diff
@@ -118,16 +119,14 @@ export class ApplicationUserControlsPopup extends React.Component {
     if (this.props.popup === "profile") {
       const topSection = (
         <div css={Styles.HORIZONTAL_CONTAINER} style={{ marginBottom: 14 }}>
-          <span
-            css={STYLES_PROFILE_IMAGE}
-            style={{
-              cursor: "default",
-              width: 46,
-              height: 46,
-              marginRight: 16,
-              backgroundImage: `url('${this.props.viewer.data.photo}')`,
-            }}
-          />
+          <div style={{ marginRight: '16px', cursor: 'default' }} >
+            <ProfilePhoto 
+              size={46}
+              userId={this.props.viewer.id}
```

Comment:
I think it's simpler if we just pass in a prop `user` rather than having to pick out the user id and user photo. Since there's always a user object and it's always structured the same way. The ProfilePhoto component can pick out user.id and user.data.photo out of it.

This way there's less to write each time (and less room to mess up), and there's fewer things to change if we decide to change how user is structured in the future